IDEAS/Buildings/Components/InternalWall.mo

```diff
@@ -141,6 +139,8 @@ public
     if hasCavity and sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.OnePort
     "1-port model for open door"
     annotation (Placement(transformation(extent={{-10,58},{10,78}})));
+initial equation
+  QTra_design=U_value*A*(TRefZon - TRef_b);
```

Comment:
TRefZon refers to `TRefZon` in `propsBusA`. Maybe a small documentation string might be instructive for other users, explaining that `TRef_b` is linked to `propsBusB` and `TRefZon` to `propsBusA`.